Docs 1707: Technical edit of page (#1267)
MultiSafepay/docs: content/faq/getting-started/test-payment-details.md

read_more: "."
---

This page assumes you have followed the steps on [Getting started](/guides/getting-started) and are ready to test different payment methods and scenarios in your [MultiSafepay Test Control](https://testMerchant.MultiSafepay.com/).

It provides information about test credentials, sample statuses, possible errors, and valid payment pages.

> **Note:** You cannot process test refunds. 

Reviewer: ```suggestion
**Note:** By default, you cannot process test refunds. To enable outgoing payments, email the Integration Team at <integration@multisafepay.com>  
```

## Banks

> ### AfterPay

Reviewer: ```suggestion
```

Reviewer: This is a duplicate

> To enable AfterPay in your MultiSafepay Test Control, email the Integration Team at <integration@multisafepay.com>

Reviewer: ```suggestion
```

### Apple Pay

To test Apple Pay, see [Compatibility and testing](/payment-methods/wallet/applepay/#compatibility-and-testing).

### Bancontact

Test card number: See the table below.

Possible errors: The test QR codes can only be read with a general QR code application. If you scan the code using the Bancontact app, an error occurs.

Sample statuses:

| Card number| Status    | Description              |
| ---------| --------- | ------------------------ |
| 67034500054620008 | Completed | Transaction was completed (3D enrolled) |
| 67034500054610009| Declined  | Transaction was declined (card must be 3D enrolled) |
| 67039902990000045| Declined  | Transaction was declined (3D authentication failed) |
| 67039902990000011| Declined  | Transaction was declined (3D authentication successful, but insufficient funds) |

### Banktransfer
Test IBANs: See the table below.

Sample statuses:

| IBAN | Status    | Description              |
| ---------| --------- | ------------------------ |
| NL87ABNA0000000002| Initialized/ Declined | Transaction is initialized. After 1 minute, this changes to declined. |

> | NL87ABNA0000000003| Initialized/ Uncleared/ Completed | Transaction is initialized. After 1 minute, this changes to Uncelared. After 1 more minute, it changes to Completed. |

Reviewer: ```suggestion
| NL87ABNA0000000003| Initialized/ Uncleared/ Completed | Transaction is initialized. After 1 minute, this changes to Uncleared. After 1 more minute, it changes to Completed. |
```

| Declined  | Transaction was declined |

### iDEAL

Sample statuses:

| Status                | Description              |
| --------------------- | ------------------------ |
| Completed             | Transaction was completed |
| Declined              | Transaction was declined |
| Cancelled             | Transaction was cancelled |
| Initialized/ Completed | Transaction is initialized. After 1 minute, this changes to Completed. |
| Initialized/ Declined  | Transaction is initialized. After 1 minute, this changes to Declined. |

### iDEAL QR

You cannot test iDEAL QR in your MultiSafepay Test Control. You can only make test payments in your live MultiSafepay Control.

### ING Home'Pay 

Sample statuses:

 Status    | Description              |
| --------- | ------------------------ |
| Completed | Transaction was completed |
| Cancelled | Transaction was cancelled |

### KBC

Payment page: KBC can only be processed on our new payment page starting with PayV2.

Sample statuses:

| Status    | Description              |
| --------- | ------------------------ |
| Completed | Transaction was completed |
| Cancelled | Transaction is void / cancelled |

### Request to Pay

> Testing environment: You can test Request to Pay transactions though Deutsche Bank. In the **Bank** field, select **Demo Bank**. 

Reviewer: ```suggestion
Testing environment: You can test Request to Pay transactions through Deutsche Bank. In the **Bank** field, select **Demo Bank**. 
```

Test IBANs: See the table below.

Sample statuses:

| IBAN | Status    | Description              |
| ---------| --------- | ------------------------ |
| NL87ABNA0000000001| Initialized/ Completed | Transaction is initialized. After 2 minutes, this changes to Completed. |
| NL87ABNA0000000002| Initialized/ Declined | Transaction is initialized. After 2 minutes, this changes to Declined. |
| NL87ABNA0000000003| Initialized/ Uncleared/ Completed | Transaction is initialized. After 2 minutes, this changes to Uncleared. After 1 more minute, it changes to Completed. |
| NL87ABNA0000000004| Initialized/ Uncleared/ Declined | Transaction is initialized. After 2 minutes, this changes to Uncleared. After 1 more minute, it changes to Declined. |

### SOFORT Banking

Sample statuses:

| Status    | Description              |
| --------- | ------------------------ |
| Completed | Transaction was completed |
| Cancelled | Transaction was cancelled |

### Trustly 

Sample statuses:

 Status    | Description              |
| --------- | ------------------------ |
| Completed | Transaction was completed |
| Cancelled | Transaction was cancelled |

## Billing suites

### AfterPay

> To enable AfterPay as payment method in your MultiSafepay Test Control, email the Integration Team at <integration@multisafepay.com>

Reviewer: ```suggestion
To enable AfterPay in your MultiSafepay Test Control, email the Integration Team at <integration@multisafepay.com>
```

### Betaal per Maand

> Testing environment: You cannot test Betaal per Maand in MultiSafepay Test Control. When activating Betaal per Maand as payment method in your live MultiSafepay Control, you can test it before go live.

Reviewer: ```suggestion
Testing environment: You cannot test Betaal per Maand in MultiSafepay Test Control. When activating Betaal per Maand as a payment method in your live MultiSafepay Control, you can test it before going live.
```

### Alipay

Sample statuses:

 Status    | Description              |
| --------- | ------------------------ |
| Completed | Transaction was completed |
| Cancelled | Transaction was cancelled |

### PayPal

Test credentials: [API key](/tools/multisafepay-control/get-your-api-key/)

To test PayPal transactions, follow these steps:

1. Send a [Direct or redirect](/faq/api/difference-between-direct-and-redirect/) API request.
2. The payment is processed in MultiSafepay Test Control as **Successful**, with order status **Completed**, and transaction status **Uncleared**.
3. Since MultiSafepay does not collect payments on behalf of PayPal, the financial (transaction) status remains **Initialized** and cannot be changed to **Completed**.

Sample statuses:

| Status    | Description              |
| --------- | ------------------------ |
| Completed | Transaction was completed |
| Declined | Transaction was declined |
| Initialized/ Completed | Payment blocked by PayPal, then accepted |
| Initialized/ Declined | Payment blocked by PayPal, then declined |
| Cancelled | Transaction was cancelled |

## Prepaid cards

### Gift cards

You can test Intersolve gift cards. 

> Testing environment: When activating a gift card as payment method in your live MultiSafepay Control, you can test it before go live.

Reviewer: ```suggestion
Testing environment: When activating a gift card as a payment method in your live MultiSafepay Control, you can test it before going live.
```
